src/store/getters/state-getters.ts

```diff
@@ -298,13 +298,16 @@ export const isApplicationValid = (state: StateIF): boolean => {
   return (
     getDefineCompanyStep(state).valid &&
     getAddPeopleAndRoleStep(state).valid &&
+    isCertifyValid(state) &&
     // Validate different steps for Base Companies vs Coops
-    isBaseCompany(state) ? isBaseStepsValid : isCoopStepsValid &&
-    getCertifyState(state).valid &&
-    !!getCertifyState(state).certifiedBy
+    isBaseCompany(state) ? isBaseStepsValid : isCoopStepsValid
   )
```

Comment:
Consider generalizing this (or using a function) as this UI will be used for other things than "other" and "coops"... For example, other entity types, dissolution, ...